API/RestMatch.API/RestMatch.API.Infrastructure/Repositories/RestaurantRepository.cs

```diff
@@ -0,0 +1,167 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using RestMatch.API.Domain.Interfaces;
+using RestMatch.API.Domain.Models;
+using RestMatch.API.Infrastructure.Data;
+
+namespace RestMatch.API.Infrastructure.Repositories
+{
+    public class RestaurantRepository : IRestaurantRepository
+    {
+        private readonly ApplicationDbContext _context;
+
+        public RestaurantRepository(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<ICollection<Restaurant>> GetRestaurants() =>
+            await _context.Restaurants.ToListAsync();
+
+        public async Task<Restaurant?> GetRestaurant(int id) =>
+            await _context.Restaurants.Where(r => r.Id == id)
+                .Include(r => r.Cuisines).Include(r => r.ImageUrls)
```

Comment:
use .ThenInclude() instead of second include()

Reply:
not needed